tests/manage/test_pvc_disruptive.py

```diff
@@ -50,6 +50,9 @@ def disruptive_base(self, operation_to_disrupt, resource_to_delete):
         if operation_to_disrupt == 'run_io':
             DISRUPTION_OPS.delete_resource()
 
+            # Validate that it's possible to run IO again
+            self.pod_obj.run_io(storage_type=self.storage_type, size='1G')
```

Comment:
maybe wait for results?
use short runtime

Reply:
I just checked the function, I think it changed a bit from my last review, its no longer a blocking call , so we might need that here or as a different function.

Reply:
we need this https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/resources/pod.py#L95, but for simplicity I am going to club both calls and send PR

Reply:
I will test the #390 and it should solve the issue, other tests I can set the wait=False if that is what they intend

Reply:
my pods are gone, have to destroy/redeploy :(

Reply:
I think storage_type should be 'fs' irrespective of rbd or cephfs.

Reply:
Yes, I have also noticed we have to give storage_type 'fs'  for running fio on nginx pod.